src/cli/OrganizationsCLIController.ts

```diff
@@ -19,24 +19,62 @@ export class OrganizationsCLIController extends BaseCLIController {
     this.projectController = new ProjectsCLIController(folder)
   }
 
+  public async getActiveOrganization() {
+    const stateOrg = StateManager.getFolderState(this.folder.name, KEYS.ORGANIZATION)
+    if (stateOrg) {
+      return stateOrg
+    }
+    // dvc status returns org name from the repo config if it exists, otherwise it returns from user config
+    const orgName = (await this.status()).organization
```

Comment:
when this method is called we're just using the name, we could possibly just return the name and save the cli call to fetch all orgs

Reply:
the dropdown uses display name for the label. But i can just display the name instead

Reply:
although in the dashboard we use the display name for orgs in the dropdown. Might be better to use the display name in the extension as well for consistency

Reply:
`display_name` is being used as the label and `name` is used for value of the each dropdown option [here](https://github.com/DevCycleHQ/vscode-extension/pull/102/files#diff-10a83d3464ffa423233be698b840a117a3fe0b71fcfd5813e0688a7961d29840R84). The fallback for the label is `name`. We need the call to `getAllOrganizations` to fetch all orgs so that we can have entire objects with display names

Reply:
only reason i want to use the display name is cause adam brought up a good point last week about how we should refer to things the same way on the dashboard and integrations so that people don't get confused

Reply:
Yep, we should definitely use the display name — but that's sourced from getAllOrganizations (not the active organizations). We need to call `getAllOrganizations` when building the view, but we don't necessarily need to call it to get the active organization name

Reply:
ahhh okay i see what you're saying. `getActiveOrganization` doesn't need to exist, we should just be able to get the active org from state every time as it should always be set. Will fix